go/libraries/doltcore/sqle/sqlddl_test.go

```diff
@@ -45,18 +45,18 @@ func TestCreateTable(t *testing.T) {
 	}{
 		{
 			name:          "Test create single column schema",
-			query:         "create table testTable (id int primary key comment 'tag:100')",
+			query:         "create table testTable (id int primary key)",
 			expectedTable: "testTable",
 			expectedSchema: dtestutils.CreateSchema(
-				schemaNewColumn(t, "id", 100, sql.Int32, true, schema.NotNullConstraint{})),
+				schemaNewColumn(t, "id", 4817, sql.Int32, true, schema.NotNullConstraint{})),
```

Comment:
These are kind of annoying. Think about making a comparison method for the test that ignores tags?